### PR TITLE
Update to UserCalling cmdlets

### PR DESCRIPTION
**Pull request by JensTrier**
Updated cmdlets to reference current TPM preview version. Added some more examples and explanations.

### CODE REVIEW

**Review by tseward — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

File: teams/teams-ps/teams/Get-CsUserCallingSettings.md [hunk 1]
```diff
@@ -185,6 +185,30 @@ ReceiveCalls   : True
...
+UnansweredDelay           : 00:00:20
+Delegates                 :
+Delegators                : 
+CallGroupOrder            : Simultaneous
+CallGroupTargets          : {}
+GroupMembershipDetails    :
+GroupNotificationOverride : 
+```
+
+This example shows the default settings for a user that has never changed the call forward settings via Microsoft Teams. Please note that for users with settings
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsUserCallingSettings.md:210]:
```suggestion
This example shows the default settings for a user that has never changed the call forward settings via Microsoft Teams. Note that for users with settings

```

File: teams/teams-ps/teams/Set-CsUserCallingSettings.md [hunk 2]
```diff
@@ -63,8 +63,8 @@ Set-CsUserCallingSettings -Identity <String> -GroupNotificationOverride <String>
 ## DESCRIPTION
 This cmdlet sets the call forwarding, simultaneous ringing and call group settings for the specified user.
 
-When specifying settings you need to specify all settings with a settings grouping, for instance, you can't just change a forwarding target. Instead you need to start by getting the current
-settings, making the necessary changes and then setting/writing all settings within the settings group.
+When specifying settings you need to specify all settings with a settings grouping, for instance, you can't just change a forwarding target. Instead you need to start by 
+getting the current settings, making the necessary changes and then setting/writing all settings within the settings group.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsUserCallingSettings.md:67]:
```suggestion
When specifying settings you need to specify all settings with a settings grouping, for instance, you can't just change a forwarding target. Instead, you need to start by 
getting the current settings, making the necessary changes, and then setting/writing all settings within the settings group.

```

File: teams/teams-ps/teams/Set-CsUserCallingSettings.md [hunk 3]
```diff
@@ -114,11 +114,11 @@ $gmd[[array]::IndexOf($gmd.CallGroupOwnerId,'user1@contoso.com')].NotificationSe
...
 ```
 
-This example shows how to update the call group of user1@contoso.com to add user5@contoso.com and remove user6@contoso.com. In addition the notification setting for user5@contoso.com for
-user1@contoso.com's call group is set to Banner.
+This example shows how to update the call group of user1@contoso.com to add user5@contoso.com and remove user6@contoso.com. In addition the notification setting for
+user5@contoso.com for user1@contoso.com's call group is set to Banner.
 
-The key to note here is the call group membership is defined on the object of the owner of the call group, in the above case this is user1@contoso.com. However the notification setting
-for a member for a particular call group is defined on the member. In this case user5@contoso.com.
+The key to note here is the call group membership is defined on the object of the owner of the call group, in the above case this is user1@contoso.com. However the notification
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsUserCallingSettings.md:120]:
```suggestion
The key to note here is the call group membership is defined on the object of the owner of the call group, in the above case this is user1@contoso.com. However, the notification

```

File: teams/teams-ps/teams/Set-CsUserCallingSettings.md [hunk 4]
```diff
@@ -137,8 +137,8 @@ This example shows how to remove all members of the call group.
 Set-CsUserCallingSettings -Identity user10@contoso.com -GroupMembershipDetails $gmd
 ```
 
-In this example user10@contoso.com is a member of 2 call groups: user20@contoso.com and user30@contoso.com. User10@contoso.com would like to have Banner notification for the first call group
-and Mute notification for the last one.
+In this example user10@contoso.com is a member of 2 call groups: user20@contoso.com and user30@contoso.com. User10@contoso.com would like to have Banner notification for the
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsUserCallingSettings.md:140]:
```suggestion
In this example user10@contoso.com is a member of two call groups: user20@contoso.com and user30@contoso.com. User10@contoso.com would like to have Banner notification for the 

```

File: teams/teams-ps/teams/Set-CsUserCallingSettings.md [hunk 5]
```diff
@@ -148,6 +148,21 @@ Set-CsUserCallingSettings -Identity user2@contoso.com -GroupNotificationOverride
...
+```
+
+This example shows how to set the default call forwarding settings for a user.
+
+### Example 11
+```powershell
+Set-CsUserCallingSettings -Identity user7@contoso.com -IsUnansweredEnabled $false
+```
+
+This example shows turning off unanswered call forwarding for a user. The Microsoft Teams client will show this as If unanswered Do nothing.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsUserCallingSettings.md:166]:
```suggestion
This example shows turning off unanswered call forwarding for a user. The Microsoft Teams client will show this as _If unanswered Do nothing_.

```

File: teams/teams-ps/teams/Set-CsUserCallingSettings.md [hunk 6]
```diff
@@ -380,16 +399,15 @@ and -WarningVariable. For more information, see [about_CommonParameters](https:/
...
 ## NOTES
-The cmdlet is available in Teams PowerShell module 3.0.1-preview.
+The cmdlet is available in Teams PowerShell module 3.0.1-preview or 3.2.0-preview (preferred).
 
 The specified user need to have the Microsoft Phone System license assigned.
 
-When forwarding to MyDelegates, the specified user needs to have 1 or more delegates defined that is allowed to receive calls. When forwarding to Group, the specified user needs to have 1
-or more members of the user's call group.
+When forwarding to MyDelegates, the specified user needs to have 1 or more delegates defined that is allowed to receive calls. When forwarding to Group, the specified user needs
+to have 1 or more members of the user's call group.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsUserCallingSettings.md:407]:
```suggestion
When forwarding to MyDelegates, the specified user needs to have one or more delegates defined that are allowed to receive calls. When forwarding to Group, the specified user needs
to have one or more members of the user's call group.

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @bratsche, thank you for committing my suggested changes. Kindly commit the last suggestion as well.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**